src/components/NewsWidget/NewsWidget.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useContext, useRef } from 'react';
+import Scrollbars from 'react-custom-scrollbars';
+import { ArrowLeft, ArrowRight } from 'react-feather';
+import ReactMarkdown from 'react-markdown';
+import Slider, { Settings } from 'react-slick';
+import remarkGfm from 'remark-gfm';
+import { ThemeContext } from 'styled-components';
+import Earth from 'src/assets/images/earth.png';
+import { Box } from 'src/components/Box';
+import { Loader } from 'src/components/Loader';
+import { News, useGetNews } from 'src/state/pnews/hooks';
+
+import { NewsContent, NewsDate, NewsSection, NewsTitle, SlickNext } from './styleds';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const NewsFeedSettings: Settings = {
+  dots: true,
+  infinite: true,
+  speed: 500,
+  slidesToShow: 1,
+  slidesToScroll: 1,
+  arrows: false,
+  autoplay: false,
+  autoplaySpeed: 10000,
+};
+
+interface Props {
+  boxHeight?: string;
+}
+
+const NewsWidget: React.FC<Props> = ({boxHeight = "400px"}) => {
```

Comment:
there are few linting issues in this PR, please run `npm run lint:fix` and fix it

<img width="1100" alt="Screenshot 2022-05-18 at 5 59 18 PM" src="https://user-images.githubusercontent.com/6604146/169038841-88afbc4c-b50f-4118-8d1d-881c000462c6.png">